package.json

```diff
@@ -21,10 +22,12 @@
     "query-string": "^4.3.1",
     "react": "16.0.0-alpha.6",
     "react-native": "0.44.2",
+    "react-native-actionsheet": "^2.2.0",
     "react-native-communications": "^2.2.1",
     "react-native-device-info": "^0.10.2",
     "react-native-elements": "^0.9.3",
     "react-native-htmlview": "^0.9.0",
+    "react-native-material-design-searchbar": "^1.1.4",
```

Comment:
Hey @patw0929! thanks for the quick changes!
So according to your (comment)[https://github.com/gitpoint/git-point/pull/49#issuecomment-315505647] I'm glad we should use une search bar solution that we can use in both iOS and Android. Do you have any suggestions?